modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1355,8 +1355,7 @@ void ONNXImporter::populateNet(Net dstNet)
                 CV_Assert(layer_id.find(node_proto.input(i)) == layer_id.end());
 
             String interp_mode = layerParams.get<String>("coordinate_transformation_mode");
-            CV_Assert_N(interp_mode != "tf_crop_and_resize", interp_mode != "asymmetric",
-                        interp_mode != "tf_half_pixel_for_nn");
+            CV_Assert_N(interp_mode != "tf_crop_and_resize", interp_mode != "tf_half_pixel_for_nn");
```

Comment:
What is `asymmetric` mode and do we really support it?

Reply:
https://github.com/onnx/onnx/blob/master/docs/Operators.md#resize

> if coordinate_transformation_mode is "asymmetric",
> x_original = x_resized / scale,

If I am not wrong we do support it. 

https://github.com/opencv/opencv/blob/5c7c80dd27abb24051097653823f7b8f22d8ac33/modules/dnn/src/layers/resize_layer.cpp#L78-L79

https://github.com/opencv/opencv/blob/5c7c80dd27abb24051097653823f7b8f22d8ac33/modules/dnn/src/layers/resize_layer.cpp#L83-L84

Reply:
And we have test data for it in this PR, right?

Reply:
https://github.com/opencv/opencv/blob/823fc5efb8b0347aff1e945276fecccf66c95557/modules/dnn/test/test_onnx_importer.cpp#L370-L371

These two test models has `coordinate_transformation_mode` parameter value `asymmetric`
So, yes, we have test data for `asymmetric`. 

> in this PR

This test is not added in this PR but was already there. Those tests were working fine before because those subgraphs were fused to Upsample so `coordinate_transformation_mode` was not getting checked and `asymmetric` is default for Resize layer in DNN module.